os2mo_data_import/os2mo_helpers/mora_helpers.py

```diff
@@ -67,30 +67,35 @@ def _create_fieldnames(self, nodes):
             fieldnames += [str(i) + 'xsub org']
         return fieldnames
 
-    def _write_csv(self, fieldnames, rows, filename):
+    def _write_csv(self, fieldnames, rows, filename, **options):
         """ Write a csv-file from a a dataset. Only fields explicitly mentioned
         in fieldnames will be saved, the rest will be ignored.
         :param fieldnames: The headline for the columns, also act as filter.
         :param rows: A list of dicts, each list element will become a row, keys
         in dict will be matched to fieldnames.
         :param filename: The name of the exported file.
         """
+
+        if len(options.items()) > 0:
+            my_options = options
+        else:
+            my_options = {"extrasaction": "ignore",
+                          "delimiter": ";",
+                          "quoting": csv.QUOTE_ALL
+                          }
+
```

Comment:
Eventuelt kan my_options navnet elimeres fuldkomment, og vi kan bruge options hele vejen igennem.
```suggestion

        my_options = options
        my_options = my_options or {
             "extrasaction": "ignore",
             "delimiter": ";",
             "quoting": csv.QUOTE_ALL
        }
```